src/library/commissioning_session.cpp

```diff
@@ -114,8 +114,10 @@ Error CommissioningSession::SendRlyTx(const ByteArray &aDtlsMessage, bool aInclu
 
     mCommImpl.mBrClient.SendRequest(rlyTx, nullptr);
 
-    LOG_DEBUG("sent RLY_TX.ntf: CommissioningSessionState={}, joinerIID={}, length={}, includeKek={}",
-              mDtlsSession->GetStateString(), utils::Hex(GetJoinerIid()), aDtlsMessage.size(), aIncludeKek);
+    LOG_DEBUG(LOG_REGION_MESHCOP,
```

Comment:
MESHCOP seems a superset of MGMT. How about naming this region something like SESSION?

Reply:
It may be confusing to name it only `SESSION`, because there is a `Commissioner Session` connected with the BA. How about `LOG_REGION_COMMISSIONING_SESSION`? 

en... it is easy to be confused with `Commissioning Session` and `Commissioner Session`, I am thinking about renaming `commissioning_session` to `joiner_session`. Thoughts? @jwhui @simonlingoogle

Reply:
The Thread Specification differentiates as:

1. Commissioner Session: secure session between Commissioner and Border Agent.

2. Joiner Session: secure session between Commissioner and Joiner.

Reply:
Thanks, so I think we should rename `commissioning_session` (the session of commissioning a joiner) here to `joiner_session`. Agree?

Reply:
Sounds good to me!

Reply:
done.